### PR TITLE
Add GPIO control through RPC

### PR DESCRIPTION
**Pull request by SamuelYvon**
# What's new

- Add GPIO control through the RPC interface

# Verification 

- Use the protobuf definition in https://github.com/flipperdevices/flipperzero-protobuf/pull/30
- You can also use the bindings from https://github.com/flipperdevices/flipperzero_protobuf_py/pull/5
- Ensure the behavior is adequate

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/rpc/rpc_gpio.c [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        return &gpio_ext_pb3;
+    case PB_Gpio_GpioPin_PA4:
+        return &gpio_ext_pa4;
+    case PB_Gpio_GpioPin_PA6:
+        return &gpio_ext_pa6;
+    case PB_Gpio_GpioPin_PA7:
+        return &gpio_ext_pa7;
+    }
+
+    __builtin_unreachable();
```
Comment (skotopes) [inline applications/rpc/rpc_gpio.c:27]: `furi_crash("Programming Error");`

**Comment by skotopes** [comment]
We'll merge this PR after 0.60 release

**Comment by SamuelYvon** [comment]
I see 0.61 has been released ;) Do you want me to fix the merge conflicts?

**Comment by skotopes** [comment]
@SamuelYvon yep, your PR is going to be next

**Comment by skotopes** [comment]
@SamuelYvon Please update your PRs with latest dev and I'll merge it

**Comment by SamuelYvon** [comment]
@skotopes There you go :)

**Comment by skotopes** [comment]
@SamuelYvon do you have tests for this PR?

**Comment by skotopes** [comment]
Tested with https://github.com/flipperdevices/flipperzero_protobuf_py/pull/5

**Review by skotopes — APPROVED** [review]

**Comment by SamuelYvon** [comment]
> @SamuelYvon do you have tests for this PR?

No, I tested, like you did, with https://github.com/flipperdevices/flipperzero_protobuf_py/pull/5

**Comment by skotopes** [comment]
okay, we'll include it into unit tests then

**Pull request merged**